src/marketplace/items/item-details.tsx

```diff
@@ -0,0 +1,228 @@
+  },
+
+  icon: {
+    marginLeft: '20px'
+  },
+
+  header: {
+    backgroundColor: '#2a3540'
+  },
+
+  body: {
+    textAlign: 'justify',
+    margin: '20px',
+    color: '#fff',
+    fontFamily: 'ProximaNovaRegular',
+    fontSize: '16px',
+    fontWeight: 400,
+    lineHeight: 1.67
+  },
+
+  formControl: {
+    marginRight: '100px'
+  },
+
+  divider: {
+    backgroundColor: '#475768'
+  },
+
+  dividerWrapper: {
+    width: '100%'
+  },
+
+  formGroup: {
+    '& span': {
+      fontSize: '14px'
+    },
+
+    '& span strong': {
+      color: '#93b0c1',
+      fontSize: '16px',
+      paddingRight: '20px'
+    }
+  },
+
+  bullet: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '30px',
+    height: '28px',
+    borderRadius: '18px',
+    backgroundColor: '#3b4b59',
+    border: 'solid 1px #495b70',
+  },
+
+  requirement: {
+    paddingLeft: '20px',
+    color: '#93b0c1'
+  },
+
+  requirements: {
+    width: '400px',
+  },
+
+  description: {
+    marginTop: '0px'
+  }
+};
+
+export type ItemProps = {
+  name: string,
+  logo: string,
+  status: string,
+  integration: string,
+  description: string,
+  location: string,
+  year_launched: number,
+  coin_pairs: string,
+  maker_fee: string,
+  taker_fee: string,
+  fiat_payments: string,
+  fiat_supported: Array<string>,
+  margin_trading: string,
+  kyc_aml: string,
+  excluded_residents: string,
+  url: string,
+  email: string,
+  kyc_template: Array<string>,
+}
+
+export type ItemDetailsProps = {
+  item: ItemProps,
+  unlockAction?: ((event: React.MouseEvent<HTMLElement>) => void) 
+}
+
+const getKYCRequirements = (requirements: Array<string>, classes: Partial<ClassNameMap<string>>) =>{
+  return requirements.map(requirement => {
+    return (
+      <Grid item>
+        <Grid container direction='row' justify='space-between'>
+          <Grid item className={classes.bullet}>
+            <TickIcon />
+          </Grid>
+          <Grid item className={classes.requirement}>
+            {requirement}
+          </Grid>
+        </Grid>
+      </Grid>
+    );
+  });
+}
+
+const unlockActionCall = (event: React.MouseEvent<HTMLElement>, unlockAction: ((event: React.MouseEvent<HTMLElement>) => void) | undefined, item: ItemProps) => {
+  if(item.status !== 'Active' || !unlockAction) {
+    return;
+  }
+
+  unlockAction(event);
+}
+
+export const ItemDetails = injectSheet(styles)<ItemDetailsProps>(({classes, children, item, unlockAction}) => (
+  <Grid container className={classes.root}>
+    <Grid item>
+      <Grid container id='header' direction='row' justify='flex-start' alignItems='center' className={classes.header}>
+        <Grid item id='icon' className={classes.icon}>
+          <img src={item.logo}/>
+        </Grid>
+        <Grid item id='title' className={classes.title}>
+          <H2>{item.name}</H2>
+        </Grid>
+      </Grid>
+      <Grid item id='body' className={classes.body}>
```

Comment:
I don't think setting id is appropriate, will there never be multiple ItemDetails on a page? Or it means something else in context of material ui?

Reply:
@sk91 I think we can use the id for e2e.